chain-signatures/node/src/indexer.rs

```diff
@@ -146,7 +146,7 @@ impl Indexer {
         block_height: BlockHeight,
         gcp: &GcpService,
     ) -> Result<(), DatastoreStorageError> {
-        tracing::trace!(block_height, "update_block_height");
+        tracing::debug!(block_height, "update_block_height");
```

Comment:
It looks like this and many other logs do not have a span. Do we want to add it everywhere and in the indexer in particular?

Reply:
It's going to have a target in the log to filter on. We don't use span at all.